paymentintent.go

```diff
@@ -172,10 +172,12 @@ type PaymentIntentPaymentMethodDataParams struct {
 	AUBECSDebit    *PaymentMethodAUBECSDebitParams `form:"au_becs_debit"`
 	BillingDetails *BillingDetailsParams           `form:"billing_details"`
 	Card           *PaymentMethodCardParams        `form:"card"`
+	EPS            *PaymentMethodEPSParams         `form:"eps"`
```

Comment:
Arg, it looks like this should be "EPS", but it's called `Eps` on `Charge`. We should probably keep it the same between the two just for the sake of consistency.

Reply:
It's Eps on Charge, but unfortunately it is also EPS in paymentmethod.go and account.go and I think it is probably better to be consistent with that?
ptal @brandur

Reply:
> It's Eps on Charge, but unfortunately it is also EPS in paymentmethod.go and account.go and I think it is probably better to be consistent with that?

Ah, fair. Yep, let's go with `EPS` then.